soql-standalone-parser/src/main/scala/com/socrata/soql/ast/Select.scala

```diff
@@ -42,11 +42,23 @@ case class JoinTable(tableName: TableName) extends JoinSelect {
 }
 case class JoinQuery(selects: BinaryTree[Select], definiteAlias: String) extends JoinSelect {
   val alias = Some(definiteAlias)
-  override def toString = "(" + Select.toString(selects) + ") AS @" + TableName.removeValidPrefix(definiteAlias)
+
+  override def toString = {
+    if (selects.inParen) {
+      Select.toString(selects) + " AS @" + TableName.removeValidPrefix(definiteAlias)
```

Comment:
This looks switched around too

Reply:
It is by design.  JoinQuery is on top of (not part of) the binaryTree.  If the binaryTree has paren, JoinQuery doesn't need another paren.